pkgs/tools/networking/linux-router/default.nix

```diff
@@ -0,0 +1,102 @@
+# SOURCE: https://github.com/NixOS/nixpkgs/pull/92787/files
+
+{ stdenv, lib, pkgs, fetchFromGitHub
+
+# Dependencies
+, bash , procps , iproute2 , dnsmasq , iptables
+
+# WiFi hotspot dependencies
+, hostapd , iw
+, wirelesstools ? null # for iwconfig
+, haveged ? null
+, qrencode ? null
+
+# General dependencies
+, coreutils , flock , gawk , getopt , gnugrep , gnused , which
+, networkmanager ? null
+
+# To fall back to haveged if entropy is low.
+#
+# Defaulting to false because not having it does not break things.
+# If it is really needed, warnings will be logged to journal.
+, useHaveged ? false
+
+# nmcli is not required for create_ap.
+# (defaulting to true because it is very likely already present)
+, useNetworkManager ? true
+
+# You only need this if 'iw' can not recognize your adapter.
+, useWirelessTools ? true
+
+# You only need this if you wish to show WiFi QR codes in terminal
+, useQrencode ? true
+}:
+
+assert useHaveged        -> !isNull haveged;
+assert useNetworkManager -> !isNull networkmanager;
+assert useWirelessTools  -> !isNull wirelesstools;
+assert useQrencode       -> !isNull qrencode;
+
+with lib;
+
+let
+  binPath = makeBinPath ([ iw procps hostapd iproute2 getopt bash dnsmasq
+    iptables coreutils which flock gnugrep gnused gawk ]
+    ++ optional useHaveged        haveged
+    ++ optional useNetworkManager networkmanager
+    ++ optional useWirelessTools  wirelesstools
+    ++ optional useQrencode       qrencode);
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "linux-router";
+  version = "0.6.2";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "garywill";
+    repo = "linux-router";
+    rev = "${version}";
+    sha256 = "193bnlwmjxsk0cri6xdylf218qayldn02pdnppvbd39ls361776z";
+  };
+
+  nativeBuildInputs = [ pkgs.makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin/ $out/.bin-wrapped
+    mv lnxrouter $out/.bin-wrapped/lnxrouter
+    makeWrapper $out/.bin-wrapped/lnxrouter $out/bin/lnxrouter --prefix PATH : ${binPath}
+  '';
+
+  meta = let
+      homepage = https://github.com/garywill/linux-router;
```

Comment:
```suggestion
      homepage = "https://github.com/garywill/linux-router";
```

Reply:
Could you please explain the difference? Will the version without quotes download the page and put it in the store?

Reply:
See here: https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md

Reply:
Thanks!